test/dca/strategies/coinBluechip/coinBluechip.test.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from "hardhat"
+import { deployUUPSUpgradeableContract } from "../../../../scripts/helper/contract"
+import { getTokenContract } from "../../../../scripts/helper/helper"
+import { currentBlockchainTime, testDcaStrategy } from "../Strategy.test"
+import { depositEthHoldBtcConfig } from "./config/depositEthHoldBtcConfig"
+import { depositUsdcHoldBtcConfig } from "./config/depositUsdcHoldBtcConfig"
+import { depositUsdcHoldEthConfig } from "./config/depositUsdcHoldEthConfig"
+
+getTokenContract
```

Comment:
What are these L9~L12 for?

Reply:
ahhh that one... Whenever I comment out some code Visual Studio Code just deletes all the unused import. This happens without any plugins, and with explicitely telling Visual Studio not to do this... Does it happen to you too?

Reply:
Maybe you can check if this exists in `settings.json`?
```
"editor.codeActionsOnSave": {
    "source.organizeImports": true
  }
```

I manually call the option when I need it instead of let it always be called on saving. Because of the same reason that is suffering you :( In Windows, a shortcut for it is alt + shift + o.